praw/models/reddit/subreddit.py

```diff
@@ -585,40 +585,32 @@ def _parse_xml_response(self, response):
             code, message, actual, maximum_size = [element.text for element in root[:4]]
             raise TooLargeMediaException(int(maximum_size), int(actual))
 
-    def _submit_media(self, data, timeout, without_websockets):
+    def _submit_media(self, data, timeout, without_websockets, websocket_url):
```

Comment:
Can we use `websocket_url=None` to indicate to not use the websocket and do away with the `without_websockets` param?

Reply:
Sounds good. Pushing this change now.